src/app/services/article.service.ts

```diff
@@ -84,68 +78,39 @@ export class ArticleService {
 
 
   async getArticleById(articleId: string) {
```

Comment:
And the realated updates here...
```suggestion
  getArticleRefById(articleId: string) {
```